packages/create-sitecore-jss/src/templates/node-headless-ssr-experience-edge/.vscode/extensions.json

```diff
@@ -4,13 +4,8 @@
     "ms-azuretools.vscode-docker",
     "dbaeumer.vscode-eslint",
     "esbenp.prettier-vscode",
-    "rvest.vs-code-prettier-eslint",
-    "eamodio.gitlens",
     "ms-vscode.PowerShell",
     "mikestead.dotenv",
-    "davidanson.vscode-markdownlint",
-    "wix.vscode-import-cost",
     "redhat.vscode-yaml",

```

Comment:
Could remove redhat.vscode-yaml from node-headless-* templates

Reply:
Do we need to have _GraphQL.vscode-graphql_ for headless templates? I guess no

Reply:
By the way, we don't really need to have _eslint_, _prettier_ extensions for *headless apps, but I decided to leave
and will create for us item in inbox to add eslint, prettier for these samples

Reply:
You're right, we don't need GraphQL.vscode-graphql for headless templates either. Let's leave eslint and prettier as you mentioned.